Test/Framework/Graphics/Texture2DNonVisualTest.cs

```diff
@@ -102,6 +102,35 @@ public void FromStreamNotPremultiplied()
             }
         }
 
+        [TestCase]
+        public void FromMemoryStream()
```

Comment:
Rename the method for the test to something that more accurately represents the test, FromStreamWithoutSeeking

Reply:
Good catch. I've renamed it to FromStreamAtTheEnd.